_posts/2021-11-07-canary.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title: "Use a canary in your Apache Kafka coal mine"
+date: 2021-11-08
+author: paolo_patierno
+---
+
+The Strimzi cluster operator is really great to deploy your Kafka cluster on Kubernetes and handling everything related to it.
+It helps with brokers configuration, managing the security aspects, doing the needed upgrades, scaling up and down and many more.
+But ... what if you want to have more insights about how the Kafka cluster is behaving?
+What if you want to know if the Kafka brokers are operating correctly so that clients can exchange messages?
+
+This is where a new component in the Strimzi family comes into the picture ... please welcome to the canary!
```

Comment:
I think we should have either:
`Welcome to the canary!`
or 
`Please welcome the canary`

But having both `please` and `to` sounds strange